mobify/platform-scaffold
# upgrade mobify-amp-sdk to 0.2.1

Apply components to AMP SDK components

 **JIRA**: https://mobify.atlassian.net/browse/AMP-63
 **Linked PRs**: n/a

## Changes
- Apply components to AMP SDK components

## How to test-drive this PR
- `npm install`
- Run `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/potions.html)
- Check Merlins Potions PDP and PLP to make sure everything looks fine.
